tools/jarcat/main.go

```diff
@@ -161,6 +162,8 @@ func main() {
 	f.Align = opts.Zip.Align
 	f.Prefix = opts.Zip.Prefix
 
+	must(f.AddManifest(opts.Zip.MainClass, opts.Zip.ClassPath))
```

Comment:
this should still be conditional - before it only wrote it if main_class was passed, now it should only write if main_class or class_path are passed.